src/Ledger/GovernanceActions.lagda

```diff
@@ -40,26 +40,26 @@ data GovRole : Set where
   CC DRep SPO : GovRole
 
 data VDeleg : Set where
-  credVoter        : GovRole → Credential → VDeleg
-  abstainRep       : VDeleg
-  noConfidenceRep  : VDeleg
+  credVoter        : GovRole → Credential →  VDeleg
+  abstainRep       :                         VDeleg
+  noConfidenceRep  :                         VDeleg
 
 record Anchor : Set where
   field  url   : String
          hash  : DocHash
 
 data GovAction : Set where
-  NoConfidence     :                                          GovAction
-  NewCommittee     : Credential ⇀ Epoch → ℙ Credential → ℚ  → GovAction
-  NewConstitution  : DocHash → Maybe ScriptHash             → GovAction
-  TriggerHF        : ProtVer                                → GovAction
-  ChangePParams    : PParamsUpdate                          → GovAction
-  TreasuryWdrl     : (RwdAddr ⇀ Coin)                       → GovAction
-  Info             :                                          GovAction
+  NoConfidence     :                                           GovAction
```

Comment:
I'm not positive you want all the `GovAction`s to be aligned, but that's how the source code is aligned.  To get it to align the same way in the pdf as it is in the code, I've added the extra whitespace.  (Same goes for the `VDeleg`s above).